yt/visualization/plot_modifications.py

```diff
@@ -2000,6 +2019,12 @@ class ScaleCallback(PlotCallback):
         The image location of the scale bar in the plot coordinate system.
         Setting pos overrides the corner parameter.
 
+    format : string, optional
```

Comment:
can you call this something like `scalebar_text_format`? I think `format` is a little ambiguous: format of what?. Can you also add what the default is and note that both `scale` and `format` must be specified in the string and nothing else?